The Game/Assets/Scripts/Agent/Player/PlayerCheatScript.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections;
```

Comment:
take it out, this is breaking the pattern again

we want to use the update function from the managers, remember?

if you want to cheat, the add the code to the player manager script. we can always take it out later.

we want to a oid all these other random scripts that arent linked to anything, the whole point is we have a chain running from the top